tests/test_router_bgp.rb

```diff
@@ -282,9 +325,11 @@ def test_routerbgp_set_get_cluster_id
         vrf = 'default'
         bgp = RouterBgp.new(asnum)
       else
+        skip(XR_NO_VRF_SUPPORT) if node.client.api == 'gRPC'
```

Comment:
Note that this will cause the entire test to report as skipped (including the default VRF part). If this isn't supported on XR as an OS limitation, the better thing to do would be either to refactor the test into two test cases (and skip only the non-default-VRF test case) or simply do `next` here instead of `skip(reason)` to bypass this section of the loop only.